Lib/configparser.py

```diff
@@ -987,7 +1001,10 @@ def _read(self, fp, fpname):
             cur_indent_level = first_nonspace.start() if first_nonspace else 0
             if (cursect is not None and optname and
                 cur_indent_level > indent_level):
-                cursect[optname].append(value)
+                if not cursect[optname]:
+                    raise MultilineContinuationError(fpname, lineno, line)
+                else:
+                    cursect[optname].append(value)
```

Comment:
else block is superfluous here.
```suggestion
                cursect[optname].append(value)
```